site/src/types.d.ts

```diff
@@ -21,10 +21,10 @@ export interface RenderContext {
   helmetContext: HelmetData['context'] | Record<string, never>;
 }
 
-export interface Page extends RouteProps {
+export type Page = RouteProps & {
   title: string;
   badge?: 'New';
-}
+};
```

Comment:
Just curious, was this a required change?

Reply:
Let's see what happens in CI, without this change 😊

Reply:
Looks like the build is failing now. https://github.com/seek-oss/braid-design-system/actions/runs/4170111393/jobs/7218757053#step:7:354

Reply:
Sure, was curious as to why now? Is there a TS update in here, or something else. Just seemed a little random thats all

Reply:
I've deduplicated dependencies which upgraded `react-router` to latest which has the new types.

Reply:
Old: https://unpkg.com/browse/react-router@6.3.0/lib/components.d.ts#L42
New: https://unpkg.com/browse/react-router@6.4.3/dist/lib/components.d.ts#L80